<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(10, 98, 38);</string>
  </property>
  <widget class="QLineEdit" name="name">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>160</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(246, 184, 49);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="surname">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>220</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(249, 186, 50);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="city">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>400</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>30</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>250</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(249, 186, 50);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="email">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>280</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(249, 186, 50);</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(246, 184, 49);</string>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="surname_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(246, 184, 49);</string>
   </property>
   <property name="text">
    <string>Surname</string>
   </property>
  </widget>
  <widget class="QLabel" name="birth_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(246, 184, 49);</string>
   </property>
   <property name="text">
    <string>Birthday</string>
   </property>
  </widget>
  <widget class="QLabel" name="email_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(246, 184, 49);</string>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLabel" name="city_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(246, 184, 49);</string>
   </property>
   <property name="text">
    <string>City</string>
   </property>
  </widget>
  <widget class="QLabel" name="password_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(246, 184, 49);</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>520</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(249, 186, 50);</string>
   </property>
   <property name="maxLength">
    <number>8</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="repassword">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>590</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(249, 186, 50);</string>
   </property>
   <property name="maxLength">
    <number>8</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="re_password_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>590</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(246, 184, 49);</string>
   </property>
   <property name="text">
    <string>Re-Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sign_Button">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>670</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(249, 186, 50);</string>
   </property>
   <property name="text">
    <string>Sign</string>
   </property>
  </widget>
  <widget class="QLabel" name="phone_number_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(246, 184, 49);</string>
   </property>
   <property name="text">
    <string>Phone_number</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="phone_number">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>460</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(249, 186, 50);</string>
   </property>
   <property name="inputMask">
    <string>(999)999 99 99</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="birthday">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>340</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(249, 186, 50);</string>
   </property>
   <property name="minimumDate">
    <date>
     <year>1900</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="date">
    <date>
     <year>2024</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>560</y>
     <width>710</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>710</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>710</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(246, 184, 49);</string>
   </property>
   <property name="text">
    <string>(The password must be 8 characters and consist of uppercase letters, lowercase letters, numbers and special characters.)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>160</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>220</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>280</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>400</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>450</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>520</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>590</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>630</y>
     <width>300</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>270</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(246, 184, 49);</string>
   </property>
   <property name="text">
    <string>(Please enter your password again)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>340</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
